src/commands.cpp

```diff
@@ -426,3 +438,70 @@ int AOClient::genRand(int min, int max)
     return random_number;
 #endif
 }
+
+void AOClient::diceThrower(int argc, QStringList argv, RollType Type)
+{
+    QString sender_name = ooc_name;
+    int max_value = server->getDiceValue("max_value");
+    int max_dice = server->getDiceValue("max_dice");
+    int bounded_value;
+    int bounded_amount;
+    QString dice_results;
+
+    if (argc == 0)
```

Comment:
For all if and else statements: opening brace goes on the same line as the if
i.e. `if (argc == 0) {`